pyproject.toml

```diff
@@ -4,6 +4,8 @@ build-backend = "setuptools.build_meta"
 
 [[tool.mypy.overrides]]
 module = [
-    'scipy'
+    'scipy',
+    'scipy.stats',
+    'scipy.special'
```

Comment:
What are these?  If we introduce a dependency on `scipy` other than for testing, we should say so in the top-level `README.md` and doc.

Reply:
They're just telling MyPy it is OK to ignore scipy imports (scipy is completely untyped, and `mypy --strict` errors on untyped libraries)

We're still only using it in testing